.gitignore

```diff
@@ -1,3 +1,5 @@
+DS_Store
```

Comment:
should be `.DS_Store`

Reply:
I made that change but I referred to it using the wrong syntax